app/assets/javascripts/discourse/app/components/topic-map/topic-participants.gjs

```diff
@@ -0,0 +1,18 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import TopicParticipant from "discourse/components/topic-map/topic-participant";
+
+export default class TopicParticipants extends Component {
+  // prettier-ignore
+  @tracked toggledUsers = new Set(this.args.userFilters);
```

Comment:
It doesn't look like `toggledUsers` will ever be reassigned, so I think we could drop the `@tracked`?